api/src/test/java/org/openmrs/OrderGroupTest.java

```diff
@@ -0,0 +1,28 @@
+package org.openmrs;
+
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
```

Comment:
You can also get rid of these javadocs since they are not saying much, we try to keep attribution out of the code since the VCS already has it anyways.